phi/memory/manager.py

```
@@ -40,59 +40,39 @@ def update_model(self) -> None:
     def add_memory(self, memory: str) -> str:
-        """Use this function to add a memory to the database.
-        Args:
-            memory (str): The memory to be stored.
-        Returns:
-            str: A message indicating if the memory was added successfully or not.
-        """
         try:
             if self.db:
                 self.db.upsert_memory(
-                    MemoryRow(user_id=self.user_id, memory=Memory(memory=memory, input=self.input_message).to_dict())
+                    MemoryRow(user_id=self.user_id, memory=Memory(memory=memory, input=input_message).to_dict())
```

Ruff

Undefined name `input_message`

(F821)